packages/react-components/react-search/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -30,11 +50,20 @@ export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTML
     },
 
     root: {
-      ref,
+      ref: useMergedRefs(searchBoxRef, ref),
       type: 'search',
       input: {}, // defining here to have access in styles hook
+
+      disabled,
+      size,
       value,
 
+      root: {
+        ref: searchBoxRootRef,
+        onFocus,
+        onBlur,
+      },
```

Comment:
I just realized -- this should probably be defined after `inputProps` is spread so it overrides `inputProps.root`, and should spread `inputProps.root` separately within it to allow authors to pass in props to the Input's root slot (even if that should be very uncommon)